src/app/components/OnboardingButton.tsx

```diff
@@ -1,19 +1,16 @@
 import { Button, Tooltip } from "antd";
 import { BookOpen } from "phosphor-react";
-import React from "react";
-import { useSelector } from "react-redux";
-import { RootState } from "../store/store";
-import { BackendStatus } from "../types/mainSlice.types";
+import React, { useContext } from "react";
 import { useOnBoardingRef } from "../../features/onboarding/utils/useOnboardingRef";
 import { OnbrdStep } from "../../features/onboarding/types/constants";
 import { OnboardingPopup } from "../../features/onboarding/components/OnboardingPopup";
 import { componentsTexts } from "../utils/constants";
+import { OnboardingContext } from "../../features/onboarding/OnboardingProvider";
 
 export const OnboardingButton = () => {
   const [isTooltipVisible, setIsTooltipVisible] = React.useState(false);
 
-  const isOnboardingAvailable =
-    useSelector<RootState>((_state) => _state.main.backendAvailable) === BackendStatus.Accessed;
+  const { isOpen: isOnboardingOpen } = useContext(OnboardingContext);
```

Comment:
Why you've dropped previous condition? Onboarding wouldn't work correctly if backend isn't available

Reply:
I compared this disabling condition with all the other buttons, and they use only `isOnboardingOpen`, so I thought the logic should be the same. But if we need to take into account backend availability for this case, okay, I returned this condition back.

Reply:
All other places use only isOnboardingOpen because they aren't on the page when backend is unavailable on the start. Or they will be unavailable, or show server error (the are tickets for that) if server responds with an error in some other cases.